pkg/reading/issue_type.go

```diff
@@ -0,0 +1,6 @@
+package reading
+
+type IssueType struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type IssueType should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)